Formula/p/poetry.rb

```diff
@@ -56,6 +56,11 @@ class Poetry < Formula
     sha256 "80d7b1f316ebfbd429f648076d6275c877ba30ba48979de4191714a75266f0ce"
   end
 
+  resource "cryptography" do
+    url "https://files.pythonhosted.org/packages/13/9e/a55763a32d340d7b06d045753c186b690e7d88780cafce5f88cb931536be/cryptography-42.0.5.tar.gz"
+    sha256 "6fe07eec95dfd477eb9530aef5bead34fec819b3aaf6c5bd6d20565da607bfe1"
+  end
+
```

Comment:
this should move to formula dep?

Reply:
yea but it's only for linux, I think we just need to update pypi_formula_mappings